### PR TITLE
created bulk loading common issues

### PR DESCRIPTION
**Pull request by ewwollesen**
#### Summary
Added a "common issues" section to the [bulk loading](https://docs.mattermost.com/onboard/bulk-loading-data.html) page with some potential issues a person could run in to while running a bulk import.

I wasn't really sure if it should go under the troubleshooting section or not, so I left it just outside of that but if you want me to put it somewhere else just let me know.

#### Ticket Link
https://github.com/mattermost/docs/issues/3292

### CODE REVIEW

File: source/onboard/bulk-loading-common-issues.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.. _bulk-loading-common-issues:
+
+Common Issues
+---------------
+Run the bulk import command as the mattermost user. Running it as root or some other user will cause issues with file permissions on imported attachments.
+
+Ensure that `file attachments are enabled <https://docs.mattermost.com/configure/configuration-settings.html#allow-file-sharing>`__, that you have enough free space in your `file storage system <https://docs.mattermost.com/configure/configuration-settings.html#file-storage-system>`__ to support the incoming attachments, and that your `maximum file size <https://docs.mattermost.com/configure/configuration-settings.html#maximum-file-size>`__ is appropriate.
+
+If you are using a version < 5.17, extended `password requirements <https://docs.mattermost.com/configure/configuration-settings.html#password>`__ need to be disabled (no uppercase, no special characters, no numbers, and password length needs to be 5). Any passwords that do not meet the requirements will cause the user import to fail.
```
Comment (cwarnermm) [inline source/onboard/bulk-loading-common-issues.rst:9]:
```suggestion
If you are using a Mattermost version prior to v5.17, extended `password requirements <https://docs.mattermost.com/configure/configuration-settings.html#password>`__ also need to be disabled (to remove requirements of uppercase, special characters, numbers, and the password length of 5). Any passwords that do not meet the requirements will cause the user import to fail.

```

**Review by cwarnermm — COMMENTED** [review]
Thank you for this great doc update, @ewwollesen! I've provided suggestions inline for consideration. Please review to ensure that I haven't inadvertently changed what was intended with my edits.

File: source/onboard/bulk-loading-common-issues.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.. _bulk-loading-common-issues:
+
+Common Issues
+---------------
+Run the bulk import command as the mattermost user. Running it as root or some other user will cause issues with file permissions on imported attachments.
```
Comment (justinegeffen) [inline source/onboard/bulk-loading-common-issues.rst:5]:
```suggestion

Run the bulk import command as the *mattermost* user. Running it as *root* or any other user will cause issues with file permissions on imported attachments.

```

File: source/onboard/bulk-loading-common-issues.rst [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.. _bulk-loading-common-issues:
+
+Common Issues
+---------------
```
Comment (justinegeffen) [inline source/onboard/bulk-loading-common-issues.rst:4]:
```suggestion
-------------

```

**Review by jespino — APPROVED** [review]
Everything looks correct from the technical stand point. Thanks @ewwollesen! :tada:

**Comment by cwarnermm** [comment]
@ewwollesen - There are a couple of review suggestions provided. Are you open to applying these suggestions?

**Comment by ewwollesen** [comment]
@cwarnermm @justinegeffen I merged in your suggested changes. Thank you,

**Pull request merged**